requirements_dev.txt

```diff
@@ -1,4 +1,7 @@
 bump2version >= 1.0.1
 flake8 >= 4.0.1
 coverage >= 4.5.4
```

Comment:
``` shell
ERROR: Cannot install -r requirements_dev.txt (line 2) and -r requirements_dev.txt (line 7) because these package versions have conflicting dependencies.

The conflict is caused by:

    flake8 4.0.1 depends on importlib-metadata<4.3; python_version < "3.8"
    sphinx 4.5.0 depends on importlib-metadata>=4.4; python_version < "3.10"
```
`flake8` and `coverage` dependency can be removed. `Jenkins` doesn't agree with `flake8` current version and `Sphinx` current version.
Thanks.